.github/workflows/migrate.yml

```
@@ -0,0 +1,64 @@
+            pg_dump ${{ secrets.STAGING }} > backup.sql
+          fi
+
+      - name: Upload Database Backup
+        uses: actions/upload-artifact@v4
+        with:
+          name: database-backup
+          path: backup.sql
+
+      - name: Run Database Migration
+        run: |
+          pnpm prisma migrate deploy
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)